interview-questions.md

```diff
@@ -8,44 +8,44 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is a hash in Ruby?
 
-Your answer:
+Your answer: A hash in ruby is pretty much like an object and has key value pairs.  An example would be [(fruit => apple), vegetable => carrot].  You can also use colons instead of the ruby rocket to assign each value to the key.
 
-Researched answer:
+Researched answer: AFter a little research a ruby hash is a collection of unique keys and their values.  A hash is like an array but the way it determines its indexes are by the keys. Another interesting fact is that the default value for a hash is nil.
 
 2. What is a gem?
 
-Your answer:
+Your answer: A gem is a library like yarn. It is specific to ruby itself and helps with language.
 
-Researched answer:
+Researched answer: A gem is a package manager that helps with downloading from different libraries that work with the ruby programming language.  It acts like yarn and helps add dependencies on which library you are calling on.
 
 3. What is Ruby on Rails?
 
-Your answer:
+Your answer: Ruby on rails is like React.  It is a server-side application that is written with ruby instead of javascript. It uses the ruby programming language to do what javascript does for React which is to shape and use data that you are adding to the application itself.
 
-Researched answer:
+Researched answer: After some research I found that Ruby on Rails is an open-sourced fullstack framework that has all the tools to help you really design a web app using the ruby language.  It is like react in the sense it is server side web application but as stated before its written in the programming language of ruby.
```

Comment:
more content could be some websites that use the language...the differences you have noticed